### DISCUSSION TITLE
Problem with map Preview Point

### DISCUSSION THREAD

**Discussion by DevTimophobia** [ppy/osu#26061] [Q&A]
Hello! Im a beginner osu mania mapper (7k) and im having this super annoying problem with preview points, i already experienced something simular with my second map, but it solved by itself after waiting some days,

On my last map: https://osu.ppy.sh/beatmapsets/2104036#mania/4421734
The problem im having is that the preview point is located randomly on the x1.1 mp3 file, and not the one i set on the x1 diff,
I tried doing it again and updating the beatmap, with no success,
Waiting 24h didnt solve it either,
Thoughts?

**Comment by peppy** [comment]
Are you using the lazer editor to set the preview point?

**Reply by DevTimophobia** [reply]
no, just the normal osu client, not lazer, should i try that maybe?
